libraries/AP_Airspeed/AP_Airspeed_I2C.cpp

```diff
@@ -80,7 +80,7 @@ void AP_Airspeed_I2C::_collect()
 
     _measurement_started_ms = 0;
 
-    if (_dev->transfer(nullptr, 0, data, sizeof(data))) {
+    if (!_dev->read_registers(0, data, sizeof(data))) {
```

Comment:
and what may be really wrong is that I used smbus_quick() with a write bit instead of read one. It should have been a read bit, which according to the datasheet would fix it. I'll write this in the patch, if it doesn't work we go back to this solution, ok?